### DISCUSSION TITLE
How to get the default settings of a window, such as the fontsize of a textblock?

### DISCUSSION THREAD

**Discussion by IronEaterBeast** [AvaloniaUI/Avalonia#13830] [Q&A]
After searching for a while, I couldn't find the answer.

I changed the font size of many textblocks, and want to change them back, in my desktop software, I want to get the default settings.

**Comment by thevortexcloud** [comment]
The actual size is determined by the theme. You can technically access the theme styles but from memory getting the values out of it via code is not straight forwards. Instead you are better off checking the source code of the theme you are using.

**Reply by IronEaterBeast** [reply]
If my program modifies the theme, I will have to remind myself to modify the corresponding code. I still think that dynamic retrieval is simpler.

**Reply by thevortexcloud** [reply]
If you are modifying the theme by setting a style, you can just clear the style? Either in code behind or somewhere else. Or just remove it from your XAML. Styles are inherited. So it will fall back to the theme value.

If you are setting a local value you can technically clear it by using `ClearValue`

https://reference.avaloniaui.net/api/Avalonia/AvaloniaObject/C52364B5

But that seems more like a hack to me.
